ItalianTranslation_PlanetCoaster.xml

```diff
@@ -2,93 +2,93 @@
-		<entry key="AdvancedTriggerEditor_SpeakerSelectSelectedToolTip" translation="Casse del Display Selezionate al momento"/>
+		<entry key="AdvancedTriggerEditor_SequencerPopUpDescription" translation="I gruppi di effetti si innescano solo dopo il tempo di attesa specificato dal momento dell'attivazione del sequenziatore"/>
+		<entry key="AdvancedTriggerEditor_SequencerPopUpTitle" translation="La mia Sequenza"/>
+		<entry key="AdvancedTriggerEditor_SequencerScrubberTitle" translation="Attiva Sequenza"/>
+		<entry key="AdvancedTriggerEditor_SequencerTriggerTitle" translation="Sequenziatore di Inneschi"/>
+		<entry key="AdvancedTriggerEditor_Sort" translation="Filtra Tutti"/>
+		<entry key="AdvancedTriggerEditor_SortTitle" translation="Filtra tutti i gruppi e gli oggetti"/>
+		<entry key="AdvancedTriggerEditor_SortTooltip" translation="L'interfaccia filtrerà solo quando chiudi il pannello o premi il pulsante Filtra Tutti"/>
+		<entry key="AdvancedTriggerEditor_SpeakerConfirmSelection" translation="Collega altoparlanti al sequenziatore"/>
+		<entry key="AdvancedTriggerEditor_SpeakerConfirmSelectionTooManyItems" translation="Troppi altoparlanti da collegare"/>
+		<entry key="AdvancedTriggerEditor_SpeakerSelectAvailableLabel" translation="Altoparlanti Disponibili"/>
+		<entry key="AdvancedTriggerEditor_SpeakerSelectAvailableToolTip" translation="Effetti disponibili nel parco"/>
+		<entry key="AdvancedTriggerEditor_SpeakerSelectDescription" translation="Usa lo strumento Selezione Multipla per selezionare e collegare gli altoparlanti"/>
+		<entry key="AdvancedTriggerEditor_SpeakerSelectSelectedLabel" translation="Altoparlanti Selezionati"/>
+		<entry key="AdvancedTriggerEditor_SpeakerSelectSelectedListLabel" translation="Altoparlanti Selezionati"/>
+		<entry key="AdvancedTriggerEditor_SpeakerSelectSelectedToolTip" translation="Altoparlanti attualmente Selezionati"/>
 		<entry key="AdvancedTriggerEditor_SpeakerSelectSelectedValue" translation="{COUNT:separatedFloat:decimalPlaces=0}/{MAX:separatedFloat:decimalPlaces=0}"/>
-		<entry key="AdvancedTriggerEditor_SpeakerSelectSubTitle" translation="Collega Casse del Display"/>
-		<entry key="AdvancedTriggerEditor_TrackEditTriggersDescription" translation="Puoi aggiungere inneschi a punti del TRACK"/>
-		<entry key="AdvancedTriggerEditor_TrackEditTriggersTitle" translation="Modifica Sequenza Innesco"/>
-		<entry key="AdvancedTriggerEditor_TrackTriggersLabel" translation="Inneschi TRACK"/>
-		<entry key="AdvancedTriggerEditor_TriggerEventPointsValueLabel" translation="Punti Innesco: {Points:int}"/>
-		<entry key="AdvancedTriggerEditor_TriggerEventStatLabel" translation="Evento Innesco"/>
-		<entry key="AdvancedTriggerEditor_TriggerEventValueLabel" translation="Oggetti: {Items:int}"/>
+		<entry key="AdvancedTriggerEditor_SpeakerSelectSubTitle" translation="Collega Altoparlanti"/>
+		<entry key="AdvancedTriggerEditor_TrackEditTriggersDescription" translation="Puoi aggiungere gli inneschi a punti del tracciato"/>
```

Comment:
Questa parte in effetti è tradotta meglio di quella di Sieg

Reply:
Bene